Update Documentation (#633)
Deep-MI/FastSurfer: doc/overview/FLAGS.md

Here, we have also added the `--3T` flag, which tells FastSurfer to register against the 3T atlas which is only relevant for the ICV estimation (eTIV).

In the following, we give an overview of the most important options. You can view a full list of options with 

```bash
./run_fastsurfer.sh --help
```

## Required arguments
* `--sd`: Output directory \$SUBJECTS_DIR (equivalent to FreeSurfer setup --> $SUBJECTS_DIR/sid/mri; $SUBJECTS_DIR/sid/surf ... will be created).
* `--sid`: Subject ID for directory inside \$SUBJECTS_DIR to be created ($SUBJECTS_DIR/sid/...)
* `--t1`: T1 full head input (does not need to be bias corrected, global path). The network was trained with conformed images (UCHAR, 256x256x256, 0.7mm - 1mm voxels and standard slice orientation). These specifications are checked in the run_prediction.py script and the image is automatically conformed if it does not comply. Note, outputs will be in the conformed space (following the FreeSurfer standard).

## Required for Docker when running surface module

> * `--fs_license`: Path to FreeSurfer license key file (only needed for the surface module). Register (for free) at [https://surfer.nmr.mgh.harvard.edu/registration.html](https://surfer.nmr.mgh.harvard.edu/registration.html) to obtain it, if you do not have FreeSurfer installed so far. This is strictly necessary if you use Docker and want to run the surface module. For local installs, your local FreeSurfer license will automatically be used. The license file is usually located in $FREESURFER_HOME/license.txt or $FREESURFER_HOME/.license . 

Reviewer: ```suggestion
* `--fs_license`: Path to FreeSurfer license key file (needed for the surface module and, if activated, the talairach registration `--tal_reg` in the segmentation). For local installs, your local FreeSurfer license will automatically be detected (usually `$FREESURFER_HOME/license.txt` or `$FREESURFER_HOME/.license`). Use this flag if autodetection fails or if you use Docker with the surface module. To get a license, [register (for free)](https://surfer.nmr.mgh.harvard.edu/registration.html).
```

## Segmentation pipeline arguments (optional)

> * `--seg_only`: Only run FastSurferVINN to generate the full brain segmentation and skip the surface pipeline.

Reviewer: It also still runs CerebNet and HypVINN...
```suggestion
* `--seg_only`: Only run the brain segmentation pipeline and skip the surface pipeline.
```

> * `--seg_log`: Name and location for the log-file for the segmentation (FastSurferVINN). Default: $SUBJECTS_DIR/$sid/scripts/deep-seg.log

Reviewer: ```suggestion
* `--seg_log`: Name and location for the log-file for the segmentation. Default: $SUBJECTS_DIR/$sid/scripts/deep-seg.log
```

* `--device`: Select device for neural network segmentation (_auto_, _cpu_, _cuda_, _cuda:<device_num>_, _mps_), where cuda means Nvidia GPU, you can select which one e.g. "cuda:1". Default: "auto", check GPU and then CPU. "mps" is for native MAC installs to use the Apple silicon (M-chip) GPU. 
* `--asegdkt_segfile`: Name of the segmentation file, which includes the aparc+DKTatlas-aseg segmentations. Requires an ABSOLUTE Path! Default location: \$SUBJECTS_DIR/\$sid/mri/aparc.DKTatlas+aseg.deep.mgz
* `--no_cereb`: Switch off the cerebellum sub-segmentation.
* `--cereb_segfile`: Name of the cerebellum segmentation file. If not provided, this intermediate DL-based segmentation will not be stored, but only the merged segmentation will be stored (see --main_segfile <filename>). Requires an ABSOLUTE Path! Default location: \$SUBJECTS_DIR/\$sid/mri/cerebellum.CerebNet.nii.gz

> * `--no_biasfield`: Deactivate the calculation of partial volume-corrected statistics in the segmentation modules.

Reviewer: ```suggestion
* `--no_biasfield`: Deactivate the biasfield correction and calculation of partial volume-corrected statistics in the segmentation modules.
```

## Surface pipeline arguments (optional)
* `--surf_only`: only run the surface pipeline recon_surf. The segmentation created by FastSurferVINN must already exist in this case.

> * `--3T`: for Talairach registration, use the 3T atlas instead of the 1.5T atlas (which is used if the flag is not provided). This gives better (more consistent with FreeSurfer) ICV estimates (eTIV) for 3T and better Talairach registration matrices, but has little impact on standard volume or surface stats.

Reviewer: ```suggestion
* `--surf_only`: Only run the surface pipeline. The segmentation created by FastSurferVINN must already exist in this case.
* `--3T`: Only affects Talairach registration: use the 3T atlas instead of the 1.5T atlas (which is used if the flag is not provided). This gives better (more consistent with FreeSurfer) ICV estimates (eTIV) for 3T and better Talairach registration matrices, but has little impact on standard volume or surface stats.
```

* `--fstess`: Use mri_tesselate instead of marching cube (default) for surface creation (not recommended, but more similar to FreeSurfer)
* `--fsqsphere`: Use FreeSurfer default instead of novel spectral spherical projection for qsphere (also not recommended)
* `--fsaparc`: Use FS aparc segmentations in addition to DL prediction (slower in this case and usually the mapped ones from the DL prediction are fine)
* `--parallel`: Run both hemispheres in parallel

> * `--no_fs_T1`: Do not generate T1.mgz (normalized nu.mgz included in standard FreeSurfer output) and create brainmask.mgz directly from norm.mgz instead. Saves 1:30 min.

Reviewer: ```suggestion
* `--no_fs_T1`: Skip generation of `T1.mgz` (normalized `nu.mgz` included in standard FreeSurfer output) and create `brainmask.mgz` directly from `norm.mgz` instead. Saves 1:30 min.
```

Nowhere else do we add the time this saves... Also, why does this take so long? Maybe a student can look into why `T1.mgz` takes so long to generate.

> * `--no_surfreg`: Skip the surface registration (`sphere.reg`), which is generated automatically by default. To safe time, use this flag to turn this off, but sphere.reg will be needed for any cross-subject statistical analysis of thickness maps. 

Reviewer: ```suggestion
* `--no_surfreg`: Skip the surface registration (which creates `sphere.reg`) to safe time. Note, `sphere.reg` will be needed for any cross-subject statistical analysis of thickness maps, so do not use this option if you plan to perform cross-subject analysis. 
```

This is a bit redundant (duplicate right now).

  The voxel size (whether set manually or derived) determines whether the surfaces are processed with highres options (below 1mm) or not.
* `--py`: Command for python, used in both pipelines. Default: python3.10
* `--conformed_name`: Name of the file in which the conformed input image will be saved. Default location: \$SUBJECTS_DIR/\$sid/mri/orig.mgz

> * `--ignore_fs_version`: Expert Option: Avoid check for FreeSurfer version. Program will otherwise terminate if the supported version (see recon-surf.sh) is not sourced. Can be used for testing dev versions (not recommended).

Reviewer: I am not sure, if options like this should be documented here at all. Or alternatively in a completely different section.

Author: I removed it, expert options should not be here.
